src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -45,6 +46,19 @@ export type SurvivalType = {
 export class StudyViewPageStore {
 
     constructor() {
+        reaction(
+            () => this._loadedInitially,
+            (loadedInitially, reaction) => {
+                if (loadedInitially) {
+                    _.forEach(this.clinicalData.result, (obj, key) => {
```

Comment:
is this similar to defaultVisibleAttributes?

Reply:
Similar? you mean executed once initially?

Reply:
i didn't look too closely but it seemed to doing the same thing.

Reply:
`defaultVisibleAttributes` pick 20 attributes from all clinical attributes, but this block filters the attributes that needs to be show depending on the data what we got for those 20 attributes. this logic is very basic. there is much more complicated logic in current study-view to handle this which will be covered in the next releases